### PR TITLE
cloud: add compression formats

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Support the following comression formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/import-csv-files.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.csv.${compress}` format. For example, if you want to import the `trips.csv.gz` file to the `bikeshare.trips` table, you can name the file as `bikeshare.trips.csv.gz`.
```
Comment (dsdashun) [inline tidb-cloud/import-csv-files.md:26]: `${db_name}.${table_name}.00000X.csv` also supports compressed files, like `bikeshare.trips.000001.csv.gz`

Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.csv.gz` file to the `bikeshare.trips` table, you can name the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ This document describes how to import uncompressed CSV files from Amazon Simple
...
 
     > **Note:**
     >
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:29]: To achieve better performance, it is recommended to limit the size of each compressed file to 100 MiB.

Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:29]:
```suggestion
    >
    > To achieve better performance, it is recommended to limit the size of each compressed file to 100 MiB.
```

File: tidb-cloud/import-csv-files.md [hunk 3]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.csv.gz` file to the `bikeshare.trips` table, you can name the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can name the file as `bikeshare.trips.sql.gz`.
```
Comment (Frank945946) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 5]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gz`, `.lz4`, `.zstd`, `.xz`, `.lzo` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

**Review by dsdashun — APPROVED** [review]

File: tidb-cloud/import-csv-files.md [hunk 7]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 8]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 9]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 10]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing the following compressed file formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 11]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.000001.csv.${compress}` format. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you have to rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 12]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.sql.${compress}` format. For example, if you want to import the `trips.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.sql.gz`.
```
Comment (Frank945946) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 13]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you have to rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you have to rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 14]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format,${suffix} is optional and can be any number like '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```

File: tidb-cloud/import-csv-files.md
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:1]: The following content also needs to be updated. 
image

File: tidb-cloud/import-csv-files.md [hunk 15]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you have to rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:26]:
```suggestion
    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which `${suffix}` is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you need to rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-csv-files.md [hunk 16]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
...
 
     > **Note:**
     >
-    > If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs), you can keep the filenames unchanged and use the **File Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud) to import your source data to a single target table.
+    > - To achieve better performance, it is recommended to limit the size of each compressed file to 100 MiB.
+    > - If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs), you can keep the filenames unchanged and use the **File Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud) to import your source data to a single target table.
```
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:31]: Is `If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs), you can keep the filenames unchanged and use the **File Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud)` also applicable to compressed files?

File: tidb-cloud/import-csv-files.md [hunk 17]
```diff
@@ -23,10 +23,12 @@ This document describes how to import uncompressed CSV files from Amazon Simple
...
+    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you have to rename the file as `bikeshare.trips.000001.csv.gz`.
 
     > **Note:**
     >
-    > If you cannot update the CSV filenames according to the preceding rules in some cases (for example, the CSV file links are also used by your other programs), you can keep the filenames unchanged and use the **File Pattern** in [Step 4](#step-4-import-csv-files-to-tidb-cloud) to import your source data to a single target table.
+    > - To achieve better performance, it is recommended to limit the size of each compressed file to 100 MiB.
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:30]:
```suggestion
    > - You only need to compress the data files, not the database and table schema files.
    > - To achieve better performance, it is recommended to limit the size of each compressed file to 100 MiB.
```

File: tidb-cloud/import-sample-data.md [hunk 18]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`.
```
Comment (hfxsd) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database and table schema files.
```

File: tidb-cloud/import-csv-files.md
Comment (hfxsd) [reply tidb-cloud/import-csv-files.md:1]: updated

File: tidb-cloud/import-csv-files.md (hunk 16 above)
Comment (dsdashun) [reply tidb-cloud/import-csv-files.md:31]: Currently file pattern cannot apply on compressed files until we support it in our Web UI

Comment (Frank945946) [reply tidb-cloud/import-csv-files.md:31]: No,it is not available for compressed source files. @hfxsd @qiancai

File: tidb-cloud/import-csv-files.md [hunk 19]
```diff
@@ -23,10 +22,13 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
     - If a CSV file contains all data of an entire table, name the file in the `${db_name}.${table_name}.csv` format, which maps to the `${db_name}.${table_name}` table when you import the data.
     - If the data of one table is separated into multiple CSV files, append a numeric suffix to these CSV files. For example, `${db_name}.${table_name}.000001.csv` and `${db_name}.${table_name}.000002.csv`. The numeric suffixes can be inconsecutive but must be in ascending order. You also need to add extra zeros before the number to ensure all the suffixes are in the same length.
+    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which `${suffix}` is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you need to rename the file as `bikeshare.trips.000001.csv.gz`.
```
Comment (Frank945946) [inline tidb-cloud/import-csv-files.md:25]:
```suggestion
    - TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed CSV files, name the files in the `${db_name}.${table_name}.${suffix}.csv.${compress}` format, in which `${suffix}` is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.csv.gz` file to the `bikeshare.trips` table, you need to rename the file as `bikeshare.trips.000001.csv.gz`.
```

File: tidb-cloud/import-sample-data.md [hunk 20]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any number such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```
Comment (Frank945946) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```

File: tidb-cloud/import-sample-data.md [hunk 21]
```diff
@@ -29,7 +29,7 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     Fill in the following parameters:
 
-    - **Data format**: select **SQL File**.
+    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which ${suffix} is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:32]:
```suggestion
    - **Data format**: select **SQL File**. TiDB Cloud supports importing compressed files in the following formats: `.gzip`, `.gz`, `.zstd`, `.zst` and `.snappy`. If you want to import compressed SQL files, name the files in the `${db_name}.${table_name}.${suffix}.sql.${compress}` format, in which `${suffix}` is optional and can be any integer such as '000001'. For example, if you want to import the `trips.000001.sql.gz` file to the `bikeshare.trips` table, you can rename the file as `bikeshare.trips.000001.sql.gz`. Note that you only need to compress the data files, not the database or table schema files.
```

File: tidb-cloud/import-sample-data.md
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:1]: <div label="GCP"> also needs similar updates for the **Data format** field description

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**